twisted/names/test/test_tap.py

```diff
@@ -96,4 +101,21 @@ def test_recursiveConfiguration(self):
                 if isinstance(recurser, Resolver):
                     recurser._parseCall.cancel()
 
+        # On Windows, we need to use a threaded resolver, which leaves trash
+        # lying about that we can't easily clean up. This isn't a good bit of
+        # code. Sorry, abstraction gods.
```

Comment:
Maybe, instead of `This isn't a good bit of  code. Sorry, abstraction gods.` we can have:

Deferreds scheduled using `callLater` are manually stopped to have a clean reactor.


---------

I think that it would be good to have a follow up tech-debt ticket in case someone has time to look into this issue and in case this will hide a nasty bug.